app/views/pages/support.html.erb

```diff
@@ -4,63 +4,71 @@
   <%= render 'shared/breadcrumb' %>
 <% end%>
 
-<main role="main" id="content">
+<main role="main" id="main-content" class="govuk-main-wrapper">
 
-  <div class="grid-row inner">
-    <div class="column-two-thirds">
-      <h1 class='heading-large'>Support</h1>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Support",
+        margin_bottom: 6,
+        heading_level: 1,
+        font_size: "l",
+      } %>
 
-      <p>The Government Digital Service (GDS) maintains the technical service behind data.gov.uk
+      <p class="govuk-body">The Government Digital Service (GDS) maintains the technical service behind data.gov.uk
          and helps government publishers to manage and update their data.</p>
 
-      <p>It cannot</p>
+      <p class="govuk-body">It cannot</p>
       <ul class="list list-bullet">
         <li>help you find or research datasets</li>
         <li>provide personal information or medical records</li>
         <li>combine data from multiple datasets</li>
       </ul>
 
-      <p>For questions about <%= link_to "GOV.UK", "https://www.gov.uk/", class: "govuk-link" %> check the GOV.UK help pages
+      <p class="govuk-body">For questions about <%= link_to "GOV.UK", "https://www.gov.uk/", class: "govuk-link" %> check the GOV.UK help pages
          or <%= link_to "contact", "https://www.gov.uk/contact/govuk", class: "govuk-link" %> the team.</p>
```

Comment:
I don't think that the links in this sentence don't quite help the user as much as they could. I'd suggest removing the link to the GOV.UK homepage, adding a link to the GOV.UK help pages, and make the contact link make sense on it's own.
```suggestion
      <p class="govuk-body">For questions about GOV.UK check the <%= link_to "GOV.UK help pages", "https://www.gov.uk/help", class: "govuk-link" %>
         or <%= link_to "contact GOV.UK", "https://www.gov.uk/contact/govuk", class: "govuk-link" %>.</p>
```